### DISCUSSION TITLE
Ограничить проверку спеллера файлами из пиара

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/content#1752] [Разное]
В пиаре #1274 спеллер полез в [файл, который не правился в рамках пиара](https://github.com/doka-guide/content/blob/main/tools/ci-cd/index.md).

![photo_2021-09-27 21 11 40](https://user-images.githubusercontent.com/50330458/134962966-a20669ae-9dad-4def-b24a-5ae34cbd4ba8.jpeg)

Было бы круто, если бы спеллер проверял только файлы из пиара во время мёржа.

Глобальную проверку можно запускать по таймеру в мейне экшном, который бы ставил задачу на код-овнера папки, если находит ошибку.

**Comment by solarrust** [comment]
Что решим в этой дискуссии? @furtivite @igsekor

**Comment by nlopin** [comment]
Это нормально. Он полез, потому что изменился файл словаря. Если словарь не меняется, то проверка просходит только по файлам пул реквеста.

Думаю, что ок было бы зафиксить эту ошибку тут же или отдельным PR, который потом подмёржили в эту ветку

**Comment by igsekor** [comment]
@nlopin совершенно прав. При изменении словаря необходимо проверить все файлы, ибо важно понять, внесёт ли это какие-то проблемы в основную ветку.
